perception_pipeline_config:
  scene_bounds: 
    min: [-2.0, -2.0, -2.0]
    max: [2.0, 2.0, 2.0]
  voxel_props:
    cubic_size: 4.0 # in metres
    voxel_resolution: 100.0 # voxels in 'cubic_size'
    primitive_sphere_radius: 0.02 # voxels in 'cubic_size'
